### PR TITLE
Add secureCodeBox project to landscape

### PR DESCRIPTION
**Pull request by Weltraumschaf**

### CODE REVIEW

**Comment by Weltraumschaf** [comment]
Hi there 👋

I guess, the Netlify errors in the pipeline are because my fork has no rights to do things on your Netlify, right?

**Review by castrojo — CHANGES_REQUESTED** [review]

**Comment by castrojo** [comment]
Yeah looks like they timed out, let's get past the first yml check and then I'll dig into the netlify logs. Thanks!

File: landscape.yml [hunk 1]
```diff
@@ -1535,6 +1535,15 @@ landscape:
             crunchbase: https://www.crunchbase.com/organization/scribe-security
             extra:
               blog_url: https://scribesecurity.com/blog/
+          - item:
+            name: secureCodeBox
+            description: secureCodeBox is an OWASP project providing an automated and scalable open source solution that integrates multiple security scanners with a simple and lightweight interface – for continuous and automated security testing.
+            homepage_url: httpw://www.securecodebox.io/
```
[the remark this answers is not in the document]
Comment (Weltraumschaf) [reply landscape.yml]: Fixed.

**Comment by Weltraumschaf** [comment]
@castrojo is this caused by my changes? 🤔

> [406] [http://www.amplifypartners.com/](http://www.amplifypartners.com/) | Failed: Network error: Not Acceptable

**Comment by Weltraumschaf** [comment]
We don't have Crunchbase (didn't knew it until now). I simply omitted the key `crunchbase` in the YAML. Is this wrong?

**Comment by Weltraumschaf** [comment]
Ok, I learned that there is a crunchbase URL to our organisation. No idea where this came from and for what it is good for, but I added it.

**Review by castrojo — APPROVED** [review]

**Pull request merged**